plasmidseeker/1.3/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM --platform=linux/amd64 ubuntu:xenial as app
+
+# ARG sets environment variables during the build stage
+ARG PLASMIDSEEKER_VER="1.3"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="PlasmidSeeker"
+LABEL software.version=$PLASMIDSEEKER_VER
+LABEL description="A k-mer based program for the identification of known plasmids from bacterial whole genome sequencing reads."
+LABEL website="https://github.com/StaPH-B/docker-builds"
```

Comment:
Can you adjust the website so that it's the website for plasmid seeker?

Reply:
Fixed to correct website.